README.md

```diff
@@ -101,15 +97,19 @@ data Teletype m a where
 makeSem ''Teletype
 
 teletypeToIO :: Member (Embed IO) r => Sem (Teletype ': r) a -> Sem r a
-teletypeToIO = interpret $ \case
+teletypeToIO = interpret \case
```

Comment:
This innocuous change drops support for ghc <8.6. I'm not sure if we care though?

Reply:
But if this is the only thing breaking e.g. ghc 8.4.3, I'm against it.

Reply:
Funny thing is, `BlockArguments` were already in `LANGUAGE` pragma on top of the snippet. And I'm not sure if anyone starts new project with versions of GHC older than 8.6 today.

Reply:
lgtm since it's used in teletype pure anyway